projects/wave-core-new/src/lib/backend/backend.model.ts

```diff
@@ -191,15 +193,52 @@ export interface InternalDataSetIdDict {
 export interface ResultDescriptorDict {
     Vector?: {
         data_type: 'Data' | 'MultiPoint' | 'MultiLineString' | 'MultiPolygon';
-        spatial_reference: string;
+        spatial_reference: SrsString;
         columns: { [key: string]: 'Categorical' | 'Decimal' | 'Number' | 'Text' };
     };
     Raster?: {
         data_type: 'U8' | 'U16' | 'U32' | 'U64' | 'I8' | 'I16' | 'I32' | 'I64' | 'F32' | 'F64';
-        spatial_reference: string;
+        spatial_reference: SrsString;
     };
 }
 
 export interface NoDataDict {
     [key: string]: number;
 }
+
+export interface PlotDataDict {
+    plot_name: string;
+    plot_type: 'Json' | 'Chart' | 'Png';
+    data: any;
+}
+
+export interface WorkflowMetadataDict {
```

Comment:
isn't the workflowmetadata just the ResultDescriptor?

Reply:
53fbb49